src/main/java/pizza/Pizza.java

```diff
@@ -0,0 +1,34 @@
+package pizza;
+
+import java.util.List;
+
+public class Pizza {
+    private final boolean isVegetarian;
+    private final List<String> ingredients;
```

Comment:
Ingredients to najlepiej jakbyś zrobił enum z dodatkami wszystkimi jakie oni tam wymieniali.  A tutaj zrobił liste z tymi enumami :)

Reply:
nazwy w enumam po angielsku ;)